Application/EdFi.Ods.Standard/Artifacts/Metadata/ApiModel.json

```diff
@@ -63399,7 +63399,7 @@
         "schema": "edfi",
         "name": "FK_DisciplineActionStudentDisciplineIncidentAssociation_DisciplineAction"
       },
-      "cardinality": "OneToOneOrMore",
+      "cardinality": "OneToZeroOrMore",
```

Comment:
When I copied in ApiModel.json from Metadata it looks to have a few more changes compared to this one here. For example, I see `EducationOrganizationAssociationTypeDescriptor` being added in mine locally, which was in the sql file changes as well. But it's not showing that change in the PR here. Another example is `StudentAssessmentEducationOrganizationAssociation` that I do not see here but I believe should be here.
![image](https://user-images.githubusercontent.com/1013553/199839191-e75e9047-4a59-473e-bacf-07ffe40114bd.png)

Reply:
`ReportedSchoolIdentifier` also shows up as a change, along with a few more FK changes that I saw in SQL but not seen here in Apimodel.json. Could you re-run this and check to see if you see any of these additional changes?

Reply:
yes  I did . Updated it